console/src/assets/i18n/zh.json

```diff
@@ -1668,6 +1672,10 @@
     "OWNER": "所有者",
     "ID": "ID",
     "NAME": "名称",
+    "AUTHORIZATIONENDPOINT": "Authorization Endpoint",
+    "TOKENENDPOINT": "Token Endpoint",
+    "USERENDPOINT": "User Endpoint",
```

Comment:
Forgot these?

Reply:
translated chinese, would stick to the endlish terms in the other langauges